### ISSUE TITLE
Failed to activate the autocomplete-citeproc package

### ISSUE THREAD

**Issue by alexmalova** [tpoisot/autocomplete-citeproc#17]
[Enter steps to reproduce:]

1. Opened a terminal from the main directory
2. called atom

However, before doing this I was trying to install Jupiter notebook and installed python3 instead of python 2.7 that I was using before. I guess it might create the problem because before I didn't have such an issue

**Atom**: 1.38.2 x64
**Electron**: 2.0.18
**OS**: Mac OS X 10.14.4
**Thrown From**: [autocomplete-citeproc](https://github.com/tpoisot/autocomplete-citeproc) package 1.4.1

### Stack Trace

Failed to activate the autocomplete-citeproc package

```
At EPERM: operation not permitted, scandir '/Users/alexmalova/Library/Application Support/AddressBook'

Error: EPERM: operation not permitted, scandir '/Users/alexmalova/Library/Application Support/AddressBook'
    at Object.fs.readdirSync (fs.js:904:18)
    at Object.fs.readdirSync (ELECTRON_ASAR.js:608:28)
    at GlobSync._readdir (/app.asar/node_modules/glob/sync.js:288:41)
    at GlobSync._readdirInGlobStar (/app.asar/node_modules/glob/sync.js:267:20)
    at GlobSync._readdir (/app.asar/node_modules/glob/sync.js:276:17)
    at GlobSync._processReaddir (/app.asar/node_modules/glob/sync.js:137:22)
    at GlobSync._process (/app.asar/node_modules/glob/sync.js:132:10)
    at GlobSync._processGlobStar (/app.asar/node_modules/glob/sync.js:380:10)
    at GlobSync._process (/app.asar/node_modules/glob/sync.js:130:10)
    at GlobSync._processGlobStar (/app.asar/node_modules/glob/sync.js:383:10)
    at GlobSync._process (/app.asar/node_modules/glob/sync.js:130:10)
    at GlobSync._processGlobStar (/app.asar/node_modules/glob/sync.js:383:10)
    at GlobSync._process (/app.asar/node_modules/glob/sync.js:130:10)
    at /app.asar/node_modules/glob/sync.js:48:10)
    at /app.asar/node_modules/glob/sync.js:26:10)
    at CiteManager.module.exports.CiteManager.addFilesFromFolder (/packages/autocomplete-citeproc/lib/manager.coffee:123:18)
    at CiteManager.module.exports.CiteManager.initialize (/packages/autocomplete-citeproc/lib/manager.coffee:66:22)
    at /packages/autocomplete-citeproc/lib/provider.coffee:20:14)
    at Object.provide (/packages/autocomplete-citeproc/lib/main.js:26:23)
    at Package.activateServices (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3421965)
    at Package.activateNow (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3419086)
    at measure (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3418410)
    at Package.measure (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3416041)
    at activationPromise.activationPromise.Promise (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3418270)
    at new Promise (<anonymous>)
    at Package.activate (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:3418213)
    at PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:381112)
    at config.transactAsync (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:380728)
    at Config.transactAsync (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:330844)
    at PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:380678)
```

### Commands

```
```

### Non-Core Packages

```
advanced-open-file 0.16.8 
atom-latex 0.8.6 
autocomplete-citeproc 1.4.1 
autocomplete-R 0.6.0 
busy-signal 2.0.1 
gpp-compiler 3.0.7 
intentions 1.1.5 
language-latex 1.2.0 
language-markdown 0.37.0 
language-matlab 0.2.1 
language-stata 1.7.3 
linter 2.3.0 
linter-ui-default 1.7.1 
markdown-preview-plus 3.8.1 
markdown-table-editor 1.1.2 
open-unsupported-files 1.0.20 
platformio-ide-terminal 2.9.1 
project-plus 1.0.0 
script 3.18.1 
tablr 1.8.3 
```

**Comment by tpoisot** [comment]
Finally figured out what this was! Fixed in `v1.4.3`.

**Comment by tpoisot** [comment]
:white_check_mark: fixed!
